rbac/server/api/packs.py

```diff
@@ -127,10 +128,32 @@
     return json({"roles": request.json.get("roles")})
 
 
+@PACKS_BP.delete("api/packs/<pack_id>")
+@authorized()
+async def delete_pack(request, pack_id):
+    """Delete pack from NEXT
+    Args:
+        request:
+            object: request object
+
+        pack_id:
+            str: ID of pack to delete
+    """
+    pack = await packs_query.check_pack_by_pack_id(request.app.config.DB_CONN, pack_id)
+    if not pack:
+        raise ApiBadRequest(
+            "Error: Pack does not currently exist or has already been deleted."
+        )
+    pack_deletion = await packs_query.delete_pack_by_id(
```

Comment:
There needs to be logic to isolate deletion capabilities to the NEXT admins or pack owner to meet the requirements of this ticket. There is a utility function to check if the user is a NextAdmins member.